lib/assessment.md

```diff
@@ -163,5 +182,5 @@ Consider the following media queries. How wide would the screen need to be for e
 ```
 
 <!-- your answer starts here -->
-
+800px
```

Comment:
You're not too far off, but this is not the correct answer. Ask yourself: at a screen width of 800px, which media queries have 'visible' CSS code? Given that, what does CSS say the dimensions should be?